alpha/veneer/semver/semver.go

```diff
@@ -434,3 +445,43 @@ func MermaidChannelWriter(cfg declcfg.DeclarativeConfig, out io.Writer) error {
 	}
 	return nil
 }
+
+func withoutBuildMetadataConflict(versions *map[string]semver.Version) error {
+	errs := []error{}
+
+	fmt.Printf("<--> input versions is %v\n", versions)
```

Comment:
Is this for debugging reason? If it is, then remove it.

Reply:
Dangit!  Fair caught.  I was sure I'd cleared all of those out. :(